src/api/goods.ts

```
@@ -8,9 +8,13 @@ export function getAll(): Promise<Good[]> {
 }
 
 export const get5First = () => {
-  return getAll().then(goods => goods); // sort and get the first 5
+  return getAll().then(goods =>
```

This code relies on `getAll()` but `getAll` does not handle server/network errors. This violates checklist item #3: "[API] - When you make a request to the server, don't forget to handle possible errors". Consider checking `response.ok` and rejecting/throwing on non-OK responses, and also handle (or let callers handle) network errors so the UI can show an error state instead of failing silently or crashing. For example, `return fetch(API_URL).then(response => { if (!response.ok) { throw new Error(response.statusText); } return response.json(); })` (then ensure callers handle the rejection).